### DISCUSSION TITLE
Help us make the plugin better?

### DISCUSSION THREAD

**Discussion by raae** [cloudinary-devs/gatsby-transformer-cloudinary#240] [General]
We (@olavea and I) are on a mission for @colbyfayock to make a roadmap for this plugin and would like a quick chat with you about your use of Gatsby + Cloudinary. 

Raise your metaphorical hand by booking a slot to chat here https://savvycal.com/raae/cloudinary-plugins

**Comment by raae** [comment]
Nudge @amcc...

**Reply by amcc** [reply]
just booked a slot in 30 mins :)

**Reply by raae** [reply]
That was a bit fast; hope to see you book a slot Friday :D
